lib/polyfill/random_uuid.js

```diff
@@ -0,0 +1,51 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+goog.provide('shaka.polyfill.RandomUUID');
+
+goog.require('shaka.log');
+goog.require('shaka.polyfill');
+
+/**
+ * @summary A polyfill to provide window.crypto.randomUUID in all browsers.
+ * @export
+ */
+shaka.polyfill.RandomUUID = class {
+  /**
+   * Install the polyfill if needed.
+   * @export
+   */
+  static install() {
+    shaka.log.debug('randomUUID.install');
+
+    if (!window.crypto) {
+      shaka.log.debug(
+          'window.crypto must be available to install randomUUID polyfill.');
```

Comment:
I had to look up support for WebCrypto to realize that it goes back to Chrome 11 / Firefox 26 / IE 11.  Can you please add a comment so that some well-meaning person doesn't assume that this is a problem when reading this code in the future?

Reply:
A link to https://caniuse.com/cryptography is enough?

Reply:
Done!